pylearn2/train_extensions/best_params.py

```diff
@@ -113,6 +113,10 @@ class MonitorBasedSaveBest(TrainExtension):
     store_best_model : bool, optional
         Whether to store the best model in memory. If False (the default),
         save_path must be defined.
+    start_epoch : int, optional
+        After this specified epoch, the model will start to be saved. Setting
+        this value to a resonable value prevents the program from saving the
+        model too much times at the begining of training.
```

Comment:
this -> the
resonable -> reasonable
program -> library
much -> many
begining -> beginning